src/generic/splitter.cpp

```diff
@@ -840,6 +843,12 @@ int wxSplitterWindow::ConvertSashPosition(int sashPosition) const
     }
     else // sashPosition == 0
     {
+        int defaultPos = GetWindowSize() / 2;
+        if (m_splitMode == wxSPLIT_VERTICAL) {
+            return m_latestVerticalSashPosition != 0 ? m_latestVerticalSashPosition : defaultPos;
+        } else if (m_splitMode == wxSPLIT_HORIZONTAL) {
+            return m_latestHorizontalSashPosition != 0 ? m_latestHorizontalSashPosition : defaultPos;
+        }
```

Comment:
Minor style fix: please put braces on their own lines for consistency, i.e.
```suggestion
        if (m_splitMode == wxSPLIT_VERTICAL)
        {
            return m_lastSplitPosition.y != 0 ? m_lastSplitPosition.y : defaultPos;
        }
        else if (m_splitMode == wxSPLIT_HORIZONTAL)
        {
            return m_lastSplitPosition.x != 0 ? m_lastSplitPosition.x : defaultPos;
        }
```